stories/index.tsx

```diff
@@ -1,26 +1,21 @@
-import React, { Component } from 'react';
-import { storiesOf, action, linkTo } from '@kadira/storybook';
+import * as React from 'react';
+import { module, storiesOf, action, linkTo } from '@kadira/storybook';
 import withContext from 'recompose/withContext';
 import { connect } from 'react-redux';
 
-import Griddle from '../src/index';
-import Cell from '../src/components/Cell';
-import Row from '../src/components/Row';
-import TableBody from '../src/components/TableBody';
-import TableHeadingCell from '../src/components/TableHeadingCell';
-import TableHeading from '../src/components/TableHeading';
-import { Table } from '../src/components/Table';
-import TableContainer from '../src/components/TableContainer';
-import ColumnDefinition from '../src/components/ColumnDefinition';
-import RowDefinition from '../src/components/RowDefinition';
+import GenericGriddle, { components, ColumnDefinition, RowDefinition } from '../src/module';
+const { Cell, Row, Table, TableBody, TableHeading, TableHeadingCell, TableContainer } = components;
```

Comment:
Since the TS definitions are now aligned with the module, it's helpful to use the module here for imports. And really, this seems more aligned with real-world usage regardless.